bin/create-release

```diff
@@ -24,8 +24,8 @@ def git_generate_changelog
   end
 
   relevant_matches = matches.select do |match|
-    # ignore merges from staging; those represent DTTs
-    match['branch'] != 'staging'
+    # ignore merges from our various server branches
+    !match.nil? && !['staging', 'test', 'levelbuilder'].include?(match['branch'])
```

Comment:
This is an interesting question - do we ignore merges "from levelbuilder" because they're routine server-branch merges and so common as to be uninteresting, or do we include them because they represent legitimate content changes that may not happen in every published version?